src/widget/OktaSignIn.js

```diff
@@ -153,7 +153,7 @@ var OktaSignIn = (function () {
     var authClient = createAuthClient(authParams);
 
     var Router;
-    if (options.stateToken && !Util.isV1StateToken(options.stateToken)) {
+    if ((options.stateToken || options.proxyIdxResponse) && !Util.isV1StateToken(options.stateToken)) {
```

Comment:
I thought we want to use your initial logic: when `proxyIdxResponse` is present, we shall go to v2 regardless state token.

Reply:
@haishengwu-okta Oh I saw you gave a ship on this discussion https://github.com/okta/okta-signin-widget/pull/1500#discussion_r521740565 so i thought you agreed with what @pradeepdewda-okta suggested?